src/Makefile

```diff
@@ -44,8 +44,8 @@ SRCS := \
 	jltypes gf typemap smallintset ast builtins module interpreter symbol \
 	dlload sys init task array genericmemory staticdata toplevel jl_uv datatype \
 	simplevector runtime_intrinsics precompile jloptions mtarraylist \
-	threading scheduler stackwalk gc gc-debug gc-pages gc-stacks gc-alloc-profiler gc-page-profiler method \
-	jlapi signal-handling safepoint timing subtype rtutils gc-heap-snapshot \
+	threading scheduler stackwalk method gc-alloc-profiler gc-heap-snapshot \
```

Comment:
Does this mean that an external GC library needs to implement the heap-snapshot interface?

Reply:
Not really? This means that an external GC may act as a client of the interface exposed by the heap snapshot.

See discussion here: https://github.com/JuliaLang/julia/pull/54238#discussion_r1583764352.